lib/nerdtree/tree_dir_node.vim

```diff
@@ -256,7 +256,7 @@ endfunction
 
 " FUNCTION: TreeDirNode.getDirChildren() {{{1
 " Return a list of all child nodes from "self.children" that are of type
-" TreeDirNode. This function supports http://github.com/scrooloose/nerdtree-project-plugin.git.
+" TreeDirNode. This function supports http://github.com/preservim/nerdtree-project-plugin.git.
```

Comment:
Revert this change. The referenced project has not moved.

Reply:
Ooops! Will do.

Reply:
Done, good catch. That also happens to fix the merge conflict that was showing up as a result of the massive code style refactoring that just landed.